sunraster/meta/meta.py

```diff
@@ -0,0 +1,52 @@
+__all__ = ["RemoteSensorMetaABC", "SlitSpectrographMetaABC"]
+
+class RemoteSensorMetaABC():
+    @property
+    def detector(self):
+        pass
+    
+    @property
+    def instrument(self):
+        pass
+    
+    @property
+    def observatory(self):
+        pass
+    
+    @property
+    def processing_level(self):
+        pass
+    
+    @property
+    def rsun_meters(self):
+        pass
+    
+    @property
+    def rsun_obs(self):
+        pass
+    
+    @property
+    def dsun(self):
+        pass
+    
+    @property
+    def observer_coordinate(self):
+        pass
+    
+    @property
+    def date(self):
```

Comment:
`date` is probably not a good key, something like`observation_date` or even better `date_start` `date_end` would map better to FITS and sanity.